### DISCUSSION TITLE
馬斯克的航天壯舉

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2152] [General]
最近馬斯克的航天創舉震驚世界 他成功接住了飛回來的火箭 成功達到航天節省大量成本的壯舉 對我來說他就像在帶著人類往前進化一樣 一直在不被別人相信的情況 做出前無古人的表現 我只能說他人可能不好 但他是真牛逼 對我來說他是我的偶像也是目標 或許有一天我也能被邀請搭乘火箭去火星研究藥物 畢竟結構在不同環境是會變化的 火星的阿斯匹靈跟台灣的可不一樣呢

**Comment by Teddy111802518** [comment]
不知道大家有什麼想做的事情 但我從來不放棄去到太空這件事 你可能會問為什麼我不讀太空系之類的 因為我果然還是想要救人的命 而且現實一點 生醫領域比較容易有飯碗 太空宇宙真的有的門檻太高 但是慢慢接近也可以啦 馬斯克帶領人類科技進步 我覺得他能讓生活更美好 但要活下去才有美好生活 所以我就成為人們擁有美好生活的後盾吧

**Comment by lai1008** [comment]
我覺得瑪斯克真的很厲害，可以把自己的想法付諸真正的行動，可以像筷子一樣把火箭夾住然後成功回收它，這樣不僅可以節省大量的經費，同時也可以將更多它們的星鏈衛星放到軌道上，且不用花費過多的金錢。

**Comment by 113607514** [comment]
最近，馬斯克的航天成就震撼了全球。他成功地回收並重複使用火箭，實現了航天領域的重大突破，顯著降低了成本。對我而言，他就像是帶領人類向前邁進的先驅。即便在許多人質疑的情況下，他依然創造了前所未有的成果。雖然他的人格可能並不完美，但他無疑是真正的天才。對我來說，他既是偶像，也是目標。或許某天，我也有機會被邀請搭乘火箭前往火星，進行藥物研究。畢竟，藥物的結構會隨著環境的不同而改變。火星上的阿斯匹靈，可能和台灣的版本完全不同呢。
